config/shared/errors/errors.go

```diff
@@ -33,41 +33,39 @@ var (
-	ErrPartitionsMisaligned        = errors.New("partitions misaligned")
-	ErrAppendAndOverwrite          = errors.New("cannot set both append and overwrite to true")
-	ErrFilesystemInvalidFormat     = errors.New("invalid filesystem format")
-	ErrFilesystemNoMountPath       = errors.New("filesystem is missing mount or path")
-	ErrFilesystemMountAndPath      = errors.New("filesystem has both mount and path defined")
-	ErrUsedCreateAndMountOpts      = errors.New("cannot use both create object and mount-level options field")
-	ErrUsedCreateAndWipeFilesystem = errors.New("cannot use both create object and wipeFilesystem field")
-	ErrExt4LabelTooLong            = errors.New("filesystem labels cannot be longer than 16 characters when using ext4")
-	ErrBtrfsLabelTooLong           = errors.New("filesystem labels cannot be longer than 256 characters when using btrfs")
-	ErrXfsLabelTooLong             = errors.New("filesystem labels cannot be longer than 12 characters when using xfs")
-	ErrSwapLabelTooLong            = errors.New("filesystem labels cannot be longer than 15 characters when using swap")
-	ErrVfatLabelTooLong            = errors.New("filesystem labels cannot be longer than 11 characters when using vfat")
-	ErrFileIllegalMode             = errors.New("illegal file mode")
-	ErrNoFilesystem                = errors.New("no filesystem specified")
-	ErrBothIDAndNameSet            = errors.New("cannot set both id and name")
-	ErrLabelTooLong                = errors.New("partition labels may not exceed 36 characters")
-	ErrDoesntMatchGUIDRegex        = errors.New("doesn't match the form \"01234567-89AB-CDEF-EDCB-A98765432101\"")
-	ErrLabelContainsColon          = errors.New("partition label will be truncated to text before the colon")
-	ErrNoPath                      = errors.New("path not specified")
-	ErrPathRelative                = errors.New("path not absolute")
-	ErrDirtyPath                   = errors.New("path is not fully simplified")
-	ErrSparesUnsupportedForLevel   = errors.New("spares unsupported for arrays with a level greater than 0")
-	ErrUnrecognizedRaidLevel       = errors.New("unrecognized raid level")
-	ErrShouldNotExistWithOthers    = errors.New("shouldExist specified false with other options also specified")
-	ErrZeroesWithShouldNotExist    = errors.New("shouldExist is false for a partition and other partition(s) has start or size 0")
-	ErrPartitionsUnitsMismatch     = errors.New("cannot mix MBs and sectors within a disk")
-	ErrSizeDeprecated              = errors.New("size is deprecated; use sizeMB instead")
-	ErrStartDeprecated             = errors.New("start is deprecated; use startMB instead")
-	ErrNeedLabelOrNumber           = errors.New("a partition number >= 1 or a label must be specified")
-	ErrDuplicateLabels             = errors.New("cannot use the same partition label twice")
+	ErrFilePermissionsUnset      = errors.New("permissions unset, defaulting to 0644")
```

Comment:
not sure how the final output will look but do you need to specify: "_file_ permissions unset" and "_directory_ permissions unset"

Reply:
These are actually old error messages, the whitespace is just changing since some irrelevent ones got removed. I can go through and fix these later in another PR.

Reply:
make sense, just checking :)